### ISSUE TITLE
Date format mismatch in test on Travis.

### ISSUE THREAD

**Issue by Capstan** [java-json-tools/msg-simple#3]
This doesn't repro for me locally, but it does on [Travis](https://travis-ci.org/java-json-tools/msg-simple/builds/583149359).

```
Gradle suite > Gradle test > com.github.fge.msgsimple.bundle.MessageBundleTest.messageFormatWorks[0](At {1,time} on {1,date}, there was {2} on planet {0,number,integer}., [Ljava.lang.Object;@712ed1a6, At 12:30:00 PM on Jul 3, 2053, there was a disturbance in the Force on planet 7.) FAILED
    java.lang.AssertionError: expected [At 12:30:00 PM on Jul 3, 2053, there was a disturbance in the Force on planet 7.] but found [At 12:30:00 on 2053 Jul 3, there was a disturbance in the Force on planet 7.]
        at org.testng.Assert.fail(Assert.java:94)
        at org.testng.Assert.failNotEquals(Assert.java:494)
        at org.testng.Assert.assertEquals(Assert.java:123)
        at org.testng.Assert.assertEquals(Assert.java:176)
        at org.testng.Assert.assertEquals(Assert.java:186)
        at com.github.fge.msgsimple.bundle.MessageBundleTest.messageFormatWorks(MessageBundleTest.java:396)
```

**Comment by Capstan** [comment]
Fixed in commit 03e8f53, which pins Travis back to JDK8.

**Comment by Capstan** [comment]
And really fixed in pr #5, to make JDK8 and JDK11 agree about what formats Locale.ROOT uses.
